### ISSUE TITLE
Bosh-Init Fails with "Volume Could Not be Found Error"

### ISSUE THREAD

**Issue by Tangierz** [starkandwayne/genesis#129]
Originally posted to Codex issues, but no response since post... Please see: https://github.com/starkandwayne/codex/issues/91

**Comment by bodymindarts** [comment]
@Tangierz just a thought. There used to be an issue like this with an outdated version of bosh-init. Can you make sure you are using the latest version.

**Comment by Tangierz** [comment]
@bodymindarts - I'm believe that I've completely torn down both the AWS and build (codes/genesis) side of things, but I'll check and post something here if I find that I have some old cruft that I didn't adequately uninstall before I took my last run at this wall.  <more soon>

Thanks for the advice!

**Comment by BR0phee** [comment]
I checked on the versions of bosh and bosh-init as suggested by @bodymindarts, and here's what I found:

I'm following the progression described in the S&W [Codex](https://github.com/starkandwayne/codex/blob/master/aws.md) documentation.

I went ahead and did a scratch install of that machine, and checked the version info:

- bosh: BOSH 1.3262.4.0
- bosh-init: version 0.0.81-775439c-2015-12-09T00:36:03Z*

These are both downlevel, so I downloaded a new copy of bosh-init from bosh.io and did a "gem update" on bosh-template, bosh_cli, and bosh_common.  Now I'm seeing the following:

- bosh: BOSH 1.3262.26.0
- bosh-init: version 0.0.101-a5c1605-2017-02-17T23:26:08Z

I ran through the process again and it worked. (or at least it sure looks like it's working - I'm about half way through the bosh install now and still looking good).
